### PR TITLE
Now display win message and most recent move

### PR DESCRIPTION
**Pull request by neuromantic99**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue), Depends on # (pull request)

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

### Screenshots

> Please attach before and after screenshots of the change if applicable.
> To upload images to a PR -- simply drag and drop or copy paste.

# Checklist:

- [ ] I have run the [`pre-commit` checks](https://pre-commit.com/) with `pre-commit run --all-files` (see `CONTRIBUTING.md` instructions to set it up)
- [ ] I have run `pytest -v` and no errors are present.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I solved any possible warnings that `pytest -v` has generated that are related to my code to the best of my knowledge.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by jjshoots** [comment]
Hi @neuromantic99, could you update the description to let us know what this PR is about?

File: pettingzoo/classic/rlcard_envs/texas_holdem.py [hunk 1]
```diff
@@ -46,7 +48,12 @@ class raw_env(RLCardBase):
     def __init__(self, num_players=2):
         super().__init__("limit-holdem", num_players, (72,))
 
-    def render(self, mode="human"):
+    def render(
+        self,
+        mode="human",
+        most_recent_move: Optional[Dict] = None,
+        win_message: Optional[str] = None,
+    ):
```
Comment (jjshoots) [inline pettingzoo/classic/rlcard_envs/texas_holdem.py:56]: This shouldn't be in render to keep inline with the other environments, could it perhaps be in the init for the environment?

File: pettingzoo/utils/wrappers/base.py [hunk 2]
```diff
@@ -78,8 +78,8 @@ def unwrapped(self):
     def close(self):
         self.env.close()
 
-    def render(self, mode="human"):
-        return self.env.render(mode)
+    def render(self, mode="human", **kwargs):
+        return self.env.render(mode, **kwargs)
```
Comment (jjshoots) [inline pettingzoo/utils/wrappers/base.py:82]: There shouldn't be kwargs in render.

File: pettingzoo/utils/wrappers/order_enforcing.py [hunk 3]
```diff
@@ -55,12 +55,12 @@ def __getattr__(self, value):
                 f"'{type(self).__name__}' object has no attribute '{value}'"
             )
 
-    def render(self, mode="human"):
+    def render(self, mode="human", **kwargs):
```
Comment (jjshoots) [inline pettingzoo/utils/wrappers/order_enforcing.py:58]: Same as above.

File: pettingzoo/utils/wrappers/order_enforcing.py [hunk 4]
```diff
@@ -55,12 +55,12 @@ def __getattr__(self, value):
...
         if not self._has_reset:
             EnvLogger.error_render_before_reset()
         assert mode in self.metadata["render_modes"]
         self._has_rendered = True
-        return super().render(mode)
+        return super().render(mode, **kwargs)
```
Comment (jjshoots) [inline pettingzoo/utils/wrappers/order_enforcing.py:63]: Same as above.

**Comment by neuromantic99** [comment]
Apologies this was meant to be a PR on our fork, as you've said it's completely unsuitable for the main repo!

**Pull request closed without merging**